packages/mobile/src/identity/verification.ts

```diff
@@ -63,6 +64,30 @@ export interface AttestationCode {
   issuer: string
 }
 
+export function* checkVerification() {
+  const attestationsWrapper: AttestationsWrapper = yield call([
+    contractKit.contracts,
+    contractKit.contracts.getAttestations,
+  ])
+
+  const account: string = yield call(getConnectedUnlockedAccount)
+  const e164Number: string = yield select(e164NumberSelector)
+
+  const status: AttestationsStatus = yield call(
+    getAttestationsStatus,
+    attestationsWrapper,
+    account,
+    e164Number
+  )
+
+  if (status.isVerified) {
+    yield put(setVerificationStatus(VerificationStatus.Done))
```

Comment:
Let's not set this. It's meant to track status updates through the verification process but the user won't be doing that process.